pgxn/neon/neon--1.0--1.1.sql

```diff
@@ -0,0 +1,5 @@
+\echo Use "ALTER EXTENSION neon UPDATE TO '1.1'" to load this file. \quit
+
+CREATE FUNCTION neon_get_stat(text) RETURNS bigint
```

Comment:
Maybe we should just call the function, neon_get_lfc_stats(), returns all stats, and allow the user to SELECT the stats they want to see.

Reply:
I thought that it can be used not only for LFC, but for reporting any other neon-specific statistic.
Making this function SRF and returning all stats seems to be good idea - will change it.